package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
it's… a little unusual and confusing to have a package.json at the top level.  I think it might misrepresent what structure is underneath.

Reply:
https://prettier.io/docs/en/configuration.html could we use a `.prettierrc.json` file instead?

Reply:
yes, I think so

Reply:
I think that might make sense.

Reply:
see https://github.com/unicode-org/cldr/pull/1919